cloud/blockstore/libs/endpoints/service_endpoint_ut.cpp

```diff
@@ -351,6 +351,10 @@ Y_UNIT_TEST_SUITE(TServiceEndpointTest)
             nullptr,    // nbdDeviceFactory
             {}          // options
         );
+        endpointManager->Start();
+        Y_DEFER {
+            endpointManager->Stop();
+        };
```

Comment:
Actually, it is not needed here, just wanted to make this test consistent with similar `ShouldWaitForRestoredEndpoints` test where we create production-like scheduler and executor